### ISSUE TITLE
Not recieving anything in discord

### ISSUE THREAD

**Issue by jonakirke94** [DiscordHooks/travis-ci-discord-webhook#20]
I've tried to set up the discord webhook but can't get anything through to the dicsord server. I followed the steps in the readme.  I used my own discord server and set up the webhook so it shouldn't be a permission issue. I changed the url to the verbose version as suggested in [#16](https://github.com/DiscordHooks/travis-ci-discord-webhook/issues/16) but appears no different to me

This is my config file:
https://github.com/jonakirke94/GTL_demo/blob/master/.travis.yml

Example of failed build:
https://travis-ci.org/jonakirke94/GTL_demo/builds/525564948

Example of successful build:
https://travis-ci.org/jonakirke94/GTL_demo/builds/525563528

**Comment by k3rn31p4nic** [comment]
If the see the logs, you'll see this:
![Screenshot](https://user-images.githubusercontent.com/19631364/56865783-7a367400-69ef-11e9-9dee-240434f98fed.png)
Which means, you've probably not configured the environment variables as mentioned in the README.

**Comment by jonakirke94** [comment]
Oops, I must have deleted it while trying to get it to work. After I added it again I got the same error as in [#16](https://github.com/DiscordHooks/travis-ci-discord-webhook/issues/16): https://travis-ci.org/jonakirke94/GTL_demo/builds/525659120

After I set my git username to "Jonathan" it works properly now. Maybe the escape character error should be mentioned in the README? 

Feel free to close the issue, Thanks.
